1.15.0 breaks CI pipelines

**PHP version**: 8.1.15
**Composer version**: 2.5.2
**Description**

Composer asks if I want to trust "php-http/discovery" when 1.15.0 gets required. This breaks CI pipelines when php-http/discovery is an indirect dependency.

**How to reproduce**
composer require php-http/discovery

**Possible Solution**
Use a new major version  (2.0.0) for breaking changes.

**Additional context**
$ composer require php-http/discovery
./composer.json has been created
Running composer update php-http/discovery
Loading composer repositories with package information
Updating dependencies
Lock file operations: 1 install, 0 updates, 0 removals
  - Locking php-http/discovery (1.15.0)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 1 install, 0 updates, 0 removals
php-http/discovery contains a Composer plugin which is currently not in your allow-plugins config. See https://getcomposer.org/allow-plugins
Do you trust "php-http/discovery" to execute code and wish to enable it now? (writes "allow-plugins" to composer.json) [y,n,d,?]

> One big takeaway here is that **CI should not run interactive**, make sure you have `-n` (or `--no-interaction`) in your Composer commands.

@Seldaek we run the CI in a library, which does not contain a `composer.lock`. even when running with `COMPOSER_NO_INTERACTION=1` the CI breaks with the 1.15.0 release after `composer install` using [ramsey/composer-install](https://github.com/ramsey/composer-install)

```
2023-02-09T13:13:20.1602042Z ##[group]Run ramsey/composer-install@v2
2023-02-09T13:13:20.1602285Z with:
2023-02-09T13:13:20.1602538Z   composer-options: --prefer-dist --no-progress
2023-02-09T13:13:20.1602818Z   dependency-versions: locked
2023-02-09T13:13:20.1603032Z env:
2023-02-09T13:13:20.1603234Z   COMPOSER_PROCESS_TIMEOUT: 0
2023-02-09T13:13:20.1603472Z   COMPOSER_NO_INTERACTION: 1
2023-02-09T13:13:20.1603697Z   COMPOSER_NO_AUDIT: 1
2023-02-09T13:13:20.1603928Z   MSYS: winsymlinks:nativestrict
2023-02-09T13:13:20.1604146Z ##[endgroup]
2023-02-09T13:13:20.1621564Z ##[group]Run ${GITHUB_ACTION_PATH}/bin/php_version.sh
2023-02-09T13:13:20.1621976Z [36;1m${GITHUB_ACTION_PATH}/bin/php_version.sh[0m
2023-02-09T13:13:20.1670622Z shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
2023-02-09T13:13:20.1670884Z env:
2023-02-09T13:13:20.1671094Z   COMPOSER_PROCESS_TIMEOUT: 0
2023-02-09T13:13:20.1671337Z   COMPOSER_NO_INTERACTION: 1
2023-02-09T13:13:20.1671563Z   COMPOSER_NO_AUDIT: 1
2023-02-09T13:13:20.1671821Z   MSYS: winsymlinks:nativestrict
2023-02-09T13:13:20.1672044Z ##[endgroup]
2023-02-09T13:13:20.2700402Z ##[group]Run ${GITHUB_ACTION_PATH}/bin/should_cache.sh ""
2023-02-09T13:13:20.2700735Z [36;1m${GITHUB_ACTION_PATH}/bin/should_cache.sh ""[0m
2023-02-09T13:13:20.2752782Z shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
2023-02-09T13:13:20.2753053Z env:
2023-02-09T13:13:20.2753259Z   COMPOSER_PROCESS_TIMEOUT: 0
2023-02-09T13:13:20.2753498Z   COMPOSER_NO_INTERACTION: 1
2023-02-09T13:13:20.2753727Z   COMPOSER_NO_AUDIT: 1
2023-02-09T13:13:20.2753963Z   MSYS: winsymlinks:nativestrict
2023-02-09T13:13:20.2754195Z ##[endgroup]
2023-02-09T13:13:20.2877416Z ##[group]Run ${GITHUB_ACTION_PATH}/bin/composer_paths.sh \
2023-02-09T13:13:20.2877765Z [36;1m${GITHUB_ACTION_PATH}/bin/composer_paths.sh \[0m
2023-02-09T13:13:20.2878015Z [36;1m  "" \[0m
2023-02-09T13:13:20.2878206Z [36;1m  "" \[0m
2023-02-09T13:13:20.2878411Z [36;1m  "/usr/bin/php"[0m
2023-02-09T13:13:20.2927136Z shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
2023-02-09T13:13:20.2927392Z env:
2023-02-09T13:13:20.2927602Z   COMPOSER_PROCESS_TIMEOUT: 0
2023-02-09T13:13:20.2927847Z   COMPOSER_NO_INTERACTION: 1
2023-02-09T13:13:20.2928158Z   COMPOSER_NO_AUDIT: 1
2023-02-09T13:13:20.2928393Z   MSYS: winsymlinks:nativestrict
2023-02-09T13:13:20.2928617Z ##[endgroup]
2023-02-09T13:13:21.0569449Z ##[group]Run ${GITHUB_ACTION_PATH}/bin/cache_key.sh \
2023-02-09T13:13:21.0569792Z [36;1m${GITHUB_ACTION_PATH}/bin/cache_key.sh \[0m
2023-02-09T13:13:21.0570044Z [36;1m  "Linux" \[0m
2023-02-09T13:13:21.0570242Z [36;1m  "8.0.27" \[0m
2023-02-09T13:13:21.0570457Z [36;1m  "locked" \[0m
2023-02-09T13:13:21.0570711Z [36;1m  "--prefer-dist --no-progress" \[0m
2023-02-09T13:13:21.0571062Z [36;1m  "4c80cd3c3837a4115fc4acf235f86d78b906db8ae35a22bfaefd2c161cf61d12" \[0m
2023-02-09T13:13:21.0571356Z [36;1m  "" \[0m
2023-02-09T13:13:21.0571544Z [36;1m  "" \[0m
2023-02-09T13:13:21.0571729Z [36;1m  ""[0m
2023-02-09T13:13:21.0624040Z shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
2023-02-09T13:13:21.0624308Z env:
2023-02-09T13:13:21.0624522Z   COMPOSER_PROCESS_TIMEOUT: 0
2023-02-09T13:13:21.0624767Z   COMPOSER_NO_INTERACTION: 1
2023-02-09T13:13:21.0624993Z   COMPOSER_NO_AUDIT: 1
2023-02-09T13:13:21.0625226Z   MSYS: winsymlinks:nativestrict
2023-02-09T13:13:21.0625450Z ##[endgroup]
2023-02-09T13:13:21.0851006Z ##[group]Run actions/cache@v3
2023-02-09T13:13:21.0851227Z with:
2023-02-09T13:13:21.0851438Z   path: /home/runner/.cache/composer
2023-02-09T13:13:21.0852014Z   key: Linux-php-8.0.27-composer---prefer-dist---no-progress-locked-4c80cd3c3837a4115fc4acf235f86d78b906db8ae35a22bfaefd2c161cf61d12
2023-02-09T13:13:21.0852722Z   restore-keys: Linux-php-8.0.27-composer---prefer-dist---no-progress-locked-

2023-02-09T13:13:21.0853102Z   enableCrossOsArchive: false
2023-02-09T13:13:21.0853350Z   fail-on-cache-miss: false
2023-02-09T13:13:21.0853560Z env:
2023-02-09T13:13:21.0853760Z   COMPOSER_PROCESS_TIMEOUT: 0
2023-02-09T13:13:21.0854114Z   COMPOSER_NO_INTERACTION: 1
2023-02-09T13:13:21.0854342Z   COMPOSER_NO_AUDIT: 1
2023-02-09T13:13:21.0854566Z   MSYS: winsymlinks:nativestrict
2023-02-09T13:13:21.0854945Z   CACHE_RESTORE_KEY: Linux-php-8.0.27-composer---prefer-dist---no-progress-locked-
2023-02-09T13:13:21.0855294Z ##[endgroup]
2023-02-09T13:13:22.5542696Z Received 50331648 of 132687391 (37.9%), 47.9 MBs/sec
2023-02-09T13:13:23.3299119Z Received 132687391 of 132687391 (100.0%), 71.2 MBs/sec
2023-02-09T13:13:23.3300842Z Cache Size: ~127 MB (132687391 B)
2023-02-09T13:13:23.3325827Z [command]/usr/bin/tar -xf /home/runner/work/_temp/4b3de17b-62c4-4a6d-a356-b2119dcfde10/cache.tzst -P -C /home/runner/work/php-clxproductnet/php-clxproductnet --use-compress-program unzstd
2023-02-09T13:13:23.6253392Z Cache restored successfully
2023-02-09T13:13:23.6525482Z Cache restored from key: Linux-php-8.0.27-composer---prefer-dist---no-progress-locked-38427e49f027c02822ec9d9be88c7395feed13fcd4cc726bd118166031ff3e68
2023-02-09T13:13:23.7095007Z ##[group]Run ${GITHUB_ACTION_PATH}/bin/composer_install.sh \
2023-02-09T13:13:23.7095400Z [36;1m${GITHUB_ACTION_PATH}/bin/composer_install.sh \[0m
2023-02-09T13:13:23.7095670Z [36;1m  "locked" \[0m
2023-02-09T13:13:23.7095934Z [36;1m  "--prefer-dist --no-progress" \[0m
2023-02-09T13:13:23.7096190Z [36;1m  "" \[0m
2023-02-09T13:13:23.7096404Z [36;1m  "/usr/bin/php" \[0m
2023-02-09T13:13:23.7096650Z [36;1m  "/usr/local/bin/composer" \[0m
2023-02-09T13:13:23.7096883Z [36;1m  ""[0m
2023-02-09T13:13:23.7149686Z shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
2023-02-09T13:13:23.7149962Z env:
2023-02-09T13:13:23.7150187Z   COMPOSER_PROCESS_TIMEOUT: 0
2023-02-09T13:13:23.7150438Z   COMPOSER_NO_INTERACTION: 1
2023-02-09T13:13:23.7150671Z   COMPOSER_NO_AUDIT: 1
2023-02-09T13:13:23.7150916Z   MSYS: winsymlinks:nativestrict
2023-02-09T13:13:23.7151309Z   CACHE_RESTORE_KEY: Linux-php-8.0.27-composer---prefer-dist---no-progress-locked-
2023-02-09T13:13:23.7151666Z ##[endgroup]
2023-02-09T13:13:23.8654988Z [32mLoading composer repositories with package information[39m
2023-02-09T13:13:24.1357786Z [30;43mWarning from https://repo.packagist.com/complex-gmbh: 
2023-02-09T13:13:24.1358757Z You are using a read-only authentication token, if you're running composer update, composer require, or composer install without a lock file, no new packages will be mirrored.[39;49m
2023-02-09T13:13:25.2610849Z [32mUpdating dependencies[39m
2023-02-09T13:13:25.2805222Z [32mLock file operations: 91 installs, 0 updates, 0 removals[39m
2023-02-09T13:13:25.2806646Z   - Locking [32mbamarni/composer-bin-plugin[39m ([33m1.8.2[39m)
2023-02-09T13:13:25.2807296Z   - Locking [32mbramus/ansi-php[39m ([33m3.1[39m)
2023-02-09T13:13:25.2808730Z   - Locking [32mclue/stream-filter[39m ([33mv1.6.0[39m)
2023-02-09T13:13:25.2809315Z   - Locking [32mcomplex/clxdeployer-recipies[39m ([33m2.44.2[39m)
2023-02-09T13:13:25.2809847Z   - Locking [32mcomplex/clxfrontendcms[39m ([33m14.0[39m)
2023-02-09T13:13:25.2810536Z   - Locking [32mcomplex/legacy[39m ([33m1.2.2[39m)
2023-02-09T13:13:25.2811531Z   - Locking [32mcomplex/library-installer-plugin[39m ([33m1.5.2[39m)
2023-02-09T13:13:25.2812027Z   - Locking [32mcomplex/rocket[39m ([33mdev-CLX6_2022R02_LTS 877920a[39m)
2023-02-09T13:13:25.2812418Z   - Locking [32mcomplex/utils[39m ([33m1.5.3[39m)
2023-02-09T13:13:25.2812802Z   - Locking [32mcomposer/installers[39m ([33mv2.2.0[39m)
2023-02-09T13:13:25.2813254Z   - Locking [32mcomposer/package-versions-deprecated[39m ([33m1.11.99.5[39m)
2023-02-09T13:13:25.2813686Z   - Locking [32mdeployer/deployer[39m ([33mv6.9.0[39m)
2023-02-09T13:13:25.2814095Z   - Locking [32mdeployer/phar-update[39m ([33mv2.2.0[39m)
2023-02-09T13:13:25.2814492Z   - Locking [32mdivineomega/do-file-cache[39m ([33mv2.0.6[39m)
2023-02-09T13:13:25.2814915Z   - Locking [32mdivineomega/do-file-cache-psr-6[39m ([33mv2.0.1[39m)
2023-02-09T13:13:25.2815346Z   - Locking [32mdivineomega/password_exposed[39m ([33mv3.2.0[39m)
2023-02-09T13:13:25.2816031Z   - Locking [32mdivineomega/psr-18-guzzle-adapter[39m ([33mv1.2.0[39m)
2023-02-09T13:13:25.2816437Z   - Locking [32mfilp/whoops[39m ([33m2.14.6[39m)
2023-02-09T13:13:25.2816827Z   - Locking [32mgraham-campbell/result-type[39m ([33mv1.1.0[39m)
2023-02-09T13:13:25.2817218Z   - Locking [32mguzzlehttp/guzzle[39m ([33m7.5.0[39m)
2023-02-09T13:13:25.2817594Z   - Locking [32mguzzlehttp/promises[39m ([33m1.5.2[39m)
2023-02-09T13:13:25.2817962Z   - Locking [32mguzzlehttp/psr7[39m ([33m2.4.3[39m)
2023-02-09T13:13:25.2818317Z   - Locking [32mhashids/hashids[39m ([33m4.1.0[39m)
2023-02-09T13:13:25.2818823Z   - Locking [32mhttp-interop/http-factory-guzzle[39m ([33m1.1.1[39m)
2023-02-09T13:13:25.2819248Z   - Locking [32mion-bazan/composer-diff[39m ([33mv1.7.0[39m)
2023-02-09T13:13:25.2819650Z   - Locking [32mjaybizzle/crawler-detect[39m ([33mv1.2.113[39m)
2023-02-09T13:13:25.2820032Z   - Locking [32mknplabs/github-api[39m ([33mv3.9.0[39m)
2023-02-09T13:13:25.2820434Z   - Locking [32mlesstif/php-jira-rest-client[39m ([33m3.1.1[39m)
2023-02-09T13:13:25.2820835Z   - Locking [32mmatthiasmullie/minify[39m ([33m1.3.70[39m)
2023-02-09T13:13:25.2821234Z   - Locking [32mmatthiasmullie/path-converter[39m ([33m1.1.3[39m)
2023-02-09T13:13:25.2821617Z   - Locking [32mmonolog/monolog[39m ([33m2.9.1[39m)
2023-02-09T13:13:25.2821984Z   - Locking [32mnetresearch/jsonmapper[39m ([33mv4.1.0[39m)
2023-02-09T13:13:25.2822347Z   - Locking [32mnyholm/psr7[39m ([33m1.5.1[39m)
2023-02-09T13:13:25.2822705Z   - Locking [32monelogin/php-saml[39m ([33m3.6.1[39m)
2023-02-09T13:13:25.2823101Z   - Locking [32mparagonie/certainty[39m ([33mv2.8.2[39m)
2023-02-09T13:13:25.2823511Z   - Locking [32mparagonie/constant_time_encoding[39m ([33mv2.6.3[39m)
2023-02-09T13:13:25.2823919Z   - Locking [32mparagonie/random_compat[39m ([33mv9.99.100[39m)
2023-02-09T13:13:25.2824313Z   - Locking [32mparagonie/sodium_compat[39m ([33mv1.19.0[39m)
2023-02-09T13:13:25.2824715Z   - Locking [32mpaypal/paypal-checkout-sdk[39m ([33m1.0.2[39m)
2023-02-09T13:13:25.2825101Z   - Locking [32mpaypal/paypalhttp[39m ([33m1.0.1[39m)
2023-02-09T13:13:25.2827179Z   - Locking [32mphp-http/cache-plugin[39m ([33m1.7.5[39m)
2023-02-09T13:13:25.2827864Z   - Locking [32mphp-http/client-common[39m ([33m2.6.0[39m)
2023-02-09T13:13:25.2828468Z   - Locking [32mphp-http/discovery[39m ([33m1.15.0[39m)
2023-02-09T13:13:25.2828860Z   - Locking [32mphp-http/httplug[39m ([33m2.3.0[39m)
2023-02-09T13:13:25.2829229Z   - Locking [32mphp-http/message[39m ([33m1.13.0[39m)
2023-02-09T13:13:25.2829622Z   - Locking [32mphp-http/message-factory[39m ([33mv1.0.2[39m)
2023-02-09T13:13:25.2830227Z   - Locking [32mphp-http/multipart-stream-builder[39m ([33m1.2.0[39m)
2023-02-09T13:13:25.2830900Z   - Locking [32mphp-http/promise[39m ([33m1.1.0[39m)
2023-02-09T13:13:25.2831440Z   - Locking [32mphp-parallel-lint/php-parallel-lint[39m ([33mv1.3.2[39m)
2023-02-09T13:13:25.2831870Z   - Locking [32mphpmailer/phpmailer[39m ([33mv6.7.1[39m)
2023-02-09T13:13:25.2833747Z   - Locking [32mphpoption/phpoption[39m ([33m1.9.0[39m)
2023-02-09T13:13:25.2834560Z   - Locking [32mphpstan/phpstan[39m ([33m1.9.17[39m)
2023-02-09T13:13:25.2835064Z   - Locking [32mpimple/pimple[39m ([33mv3.5.0[39m)
2023-02-09T13:13:25.2835850Z   - Locking [32mpsr/cache[39m ([33m1.0.1[39m)
2023-02-09T13:13:25.2836503Z   - Locking [32mpsr/container[39m ([33m2.0.1[39m)
2023-02-09T13:13:25.2836979Z   - Locking [32mpsr/http-client[39m ([33m1.0.1[39m)
2023-02-09T13:13:25.2837436Z   - Locking [32mpsr/http-factory[39m ([33m1.0.1[39m)
2023-02-09T13:13:25.2837876Z   - Locking [32mpsr/http-message[39m ([33m1.0.1[39m)
2023-02-09T13:13:25.2838302Z   - Locking [32mpsr/log[39m ([33m1.1.4[39m)
2023-02-09T13:13:25.2838780Z   - Locking [32mralouphie/getallheaders[39m ([33m3.0.3[39m)
2023-02-09T13:13:25.2839225Z   - Locking [32mrdlowrey/auryn[39m ([33mv1.4.4[39m)
2023-02-09T13:13:25.2839821Z   - Locking [32mrecombee/php-api-client[39m ([33mv4.1.0[39m)
2023-02-09T13:13:25.2840288Z   - Locking [32mrector/rector[39m ([33m0.15.13[39m)
2023-02-09T13:13:25.2840719Z   - Locking [32mrmccue/requests[39m ([33mv1.8.1[39m)
2023-02-09T13:13:25.2841172Z   - Locking [32mrobrichards/xmlseclibs[39m ([33m3.1.1[39m)
2023-02-09T13:13:25.2841614Z   - Locking [32msabre/event[39m ([33m5.1.4[39m)
2023-02-09T13:13:25.2842034Z   - Locking [32msabre/uri[39m ([33m2.2.4[39m)
2023-02-09T13:13:25.2842446Z   - Locking [32msabre/xml[39m ([33m2.2.5[39m)
2023-02-09T13:13:25.2842885Z   - Locking [32mstaabm/sql-parser-util[39m ([33m1.0[39m)
2023-02-09T13:13:25.2843414Z   - Locking [32mstaabm/sysmonitor[39m ([33m1.0.4[39m)
2023-02-09T13:13:25.2843864Z   - Locking [32mstaabm/thincache[39m ([33m1.0.5[39m)
2023-02-09T13:13:25.2844311Z   - Locking [32mstomp-php/stomp-php[39m ([33m4.6.1[39m)
2023-02-09T13:13:25.2844770Z   - Locking [32msymfony/console[39m ([33mv5.4.19[39m)
2023-02-09T13:13:25.2845249Z   - Locking [32msymfony/deprecation-contracts[39m ([33mv2.5.2[39m)
2023-02-09T13:13:25.2845744Z   - Locking [32msymfony/options-resolver[39m ([33mv5.4.19[39m)
2023-02-09T13:13:25.2846227Z   - Locking [32msymfony/polyfill-ctype[39m ([33mv1.27.0[39m)
2023-02-09T13:13:25.2846736Z   - Locking [32msymfony/polyfill-intl-grapheme[39m ([33mv1.27.0[39m)
2023-02-09T13:13:25.2847255Z   - Locking [32msymfony/polyfill-intl-normalizer[39m ([33mv1.27.0[39m)
2023-02-09T13:13:25.2847764Z   - Locking [32msymfony/polyfill-mbstring[39m ([33mv1.27.0[39m)
2023-02-09T13:13:25.2848254Z   - Locking [32msymfony/polyfill-php73[39m ([33mv1.27.0[39m)
2023-02-09T13:13:25.2848725Z   - Locking [32msymfony/polyfill-php80[39m ([33mv1.27.0[39m)
2023-02-09T13:13:25.2849172Z   - Locking [32msymfony/process[39m ([33mv5.4.19[39m)
2023-02-09T13:13:25.2849672Z   - Locking [32msymfony/service-contracts[39m ([33mv1.1.2[39m)
2023-02-09T13:13:25.2850132Z   - Locking [32msymfony/string[39m ([33mv5.4.19[39m)
2023-02-09T13:13:25.2850588Z   - Locking [32msymfony/var-dumper[39m ([33mv5.4.19[39m)
2023-02-09T13:13:25.2851031Z   - Locking [32msymfony/yaml[39m ([33mv5.4.19[39m)
2023-02-09T13:13:25.2851509Z   - Locking [32msymplify/easy-coding-standard[39m ([33m11.2.8[39m)
2023-02-09T13:13:25.2851980Z   - Locking [32mtecnickcom/tcpdf[39m ([33m6.6.2[39m)
2023-02-09T13:13:25.2852420Z   - Locking [32mvlucas/phpdotenv[39m ([33mv5.5.0[39m)
2023-02-09T13:13:25.2852872Z   - Locking [32mwebmozart/assert[39m ([33m1.11.0[39m)
2023-02-09T13:13:25.2853359Z   - Locking [32mzendframework/zendframework1[39m ([33m1.12.20[39m)
2023-02-09T13:13:25.2856531Z [32mWriting lock file[39m
2023-02-09T13:13:25.2857050Z [32mInstalling dependencies from lock file (including require-dev)[39m
2023-02-09T13:13:25.2892514Z [32mPackage operations: 91 installs, 0 updates, 0 removals[39m
2023-02-09T13:13:25.2902165Z   - Downloading [32mphp-http/discovery[39m ([33m1.15.0[39m)
2023-02-09T13:13:25.4084475Z   - Installing [32mbamarni/composer-bin-plugin[39m ([33m1.8.2[39m): Extracting archive
2023-02-09T13:13:25.9063293Z ##[error]php-http/discovery contains a Composer plugin which is blocked by your allow-plugins config. You may add it to the list if you consider it safe.
You can run "composer config --no-plugins allow-plugins.php-http/discovery [true|false]" to enable it (true) or disable it explicitly and suppress this exception (false)
See https://getcomposer.org/allow-plugins
2023-02-09T13:13:25.9071296Z 
2023-02-09T13:13:25.9114935Z [33mIn PluginManager.php line 738:[39m
2023-02-09T13:13:25.9115348Z [37;41m                                                                               [39;49m
2023-02-09T13:13:25.9115816Z [37;41m  php-http/discovery contains a Composer plugin which is blocked by your allo  [39;49m
2023-02-09T13:13:25.9116417Z [37;41m  w-plugins config. You may add it to the list if you consider it safe.        [39;49m
2023-02-09T13:13:25.9117120Z [37;41m  You can run "composer config --no-plugins allow-plugins.php-http/discovery   [39;49m
2023-02-09T13:13:25.9117636Z [37;41m  [true|false]" to enable it (true) or disable it explicitly and suppress thi  [39;49m
2023-02-09T13:13:25.9118071Z [37;41m  s exception (false)                                                          [39;49m
2023-02-09T13:13:25.9118514Z [37;41m  See https://getcomposer.org/allow-plugins                                    [39;49m
2023-02-09T13:13:25.9118925Z [37;41m                                                                               [39;49m
```

using

```
==> Setup PHP
✓ PHP Installed PHP 8.0.27

==> Setup Tools
✓ composer Added composer 2.5.2
```

in a github action runner
```
Operating System
  Ubuntu
  22.04
  LTS
Runner Image
  Image: ubuntu-22.04
  Version: 20230206
Runner Image Provisioner  2.0.108
```

--- a/composer.json
+++ b/composer.json
@@ -48,7 +48,8 @@
         "test-ci": "vendor/bin/phpspec run -c phpspec.ci.yml"
     },
     "extra": {
-        "class": "Http\\Discovery\\Composer\\Plugin"
+        "class": "Http\\Discovery\\Composer\\Plugin",
+        "plugin-optional": true
     },
     "conflict": {
         "nyholm/psr7": "<1.0"

--- a/tests/install.sh
+++ b/tests/install.sh
@@ -17,7 +17,8 @@ echo "Using directory: ${BUILD_DIR}"
 mkdir -p $BUILD_DIR
 
 # Init composer
-composer req --working-dir $BUILD_DIR php-http/discovery --no-update
+composer req --working-dir $BUILD_DIR php-http/discovery --no-update --no-plugins
+composer config --working-dir $BUILD_DIR --no-plugins allow-plugins.php-http/discovery false
 
 # Argument 3 installs additional composer packages
 composer req --working-dir $BUILD_DIR $3

